synapse/models/inet.py

```diff
@@ -88,9 +88,6 @@ def norm(self, valu, oldval=None):
         except UnicodeError as e:
             self._raiseBadValu(valu)
 
-        if not fqdnre.match(valu):
```

Comment:
We already check this above. ACE encoding the FQDN is not going to make it an invalid FQDN.